test/hooks_test.dart

```diff
@@ -642,6 +642,76 @@ main() {
     group('useLayoutEffect -', () {
       testEffectHook(useLayoutEffect);
     });
+
+    group('useDebugValue -', () {
+      ReactDartFunctionComponentFactoryProxy UseDebugValueTest1;
+      ReactDartFunctionComponentFactoryProxy UseDebugValueTest2;
+      StateHook<bool> isOnline1;
+      StateHook<bool> isOnline2;
+
+      StateHook useFriendStatus(friendID) {
```

Comment:
Unused parameter - was there a reason to keep it around?

Reply:
nope lol